scripts/run_bundler.mjs

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env node
+/**
+ * # run_bundler.mjs
+ *
+ * This file allows to create JavaScript bundles for the RxPlayer through our
+ * bundlers with the right configuration.
+ *
+ * You can either run it directly as a script (run `node run_bundler.mjs -h`
+ * to see the different options) or by requiring it as a node module.
+ * If doing the latter you will obtain a function you will have to run with the
+ * right options.
+ */
+
+import * as fs from "fs";
+import * as path from "path";
+import { pathToFileURL } from "url";
+import esbuild from "esbuild";
+import getHumanReadableHours from "./utils/get_human_readable_hours.mjs";
+
+// If true, this script is called directly
```

Comment:
What does "called directly" mean ? Script is run with `node run_bundler` ?

Reply:
Yes, as opposed to imported (we also export the `runBundler` function)